__init__.py

```diff
@@ -944,6 +956,37 @@ def _stop_beep(self):
         if self._is_playing_beep():
             self.beep_process.kill()
             self.beep_process = None
+            
+    def _build_timer_display(self, idx, timer, remaining_time):
+        color_idx = 0 if idx is None else idx % 4 - 1
+        elapsed_time = timer['duration'] - remaining_time
+        remaining_time_display = self._build_time_remaining_string(
+            remaining_time
+        )
+        if datetime.now() > timer['expires']:
+            percent_elapsed = 1
+            remaining_time_display = '-' + remaining_time_display
+            timer_expd = True
+        else:
+            percent_elapsed = elapsed_time / timer['duration']
+            timer_expd = False
+            
+        remain_time_in_ms = remaining_time * 1000
+            
+        if timer['name']:
+            timer_name = timer['name']
+        else:
+            if idx is None or idx == 1:
+                timer_name = 'Timer'
+                timer_id = 1
+            else:
+                timer_name = 'Timer ' + str(idx)
+                timer_id = idx
```

Comment:
In this first conditional path we don't get a `timer_id` set. We could do something like:

```Python
        timer_id = idx or 1
        if timer['name']:
            timer_name = timer['name'].capitalize()
        elif timer_id == 1:
            timer_name = 'Timer'
        else:
            timer_name = 'Timer ' + str(timer_id)
```